### ISSUE TITLE
Invalid loader file! Error: Not found while loading bootmgfw.efi on sd card

### ISSUE THREAD

**Issue by lliamXD383** [jlobue10/SteamDeck_rEFInd#125]
Hey i install windows 11 on my sd card and i only got this on booting menu on refind and i got this error
![17304911531448436681433792056260](https://github.com/user-attachments/assets/b37a171b-759a-4745-a3a7-e4a834b9c77c)
It only works on manual booting how can i fix this problem

**Comment by jlobue10** [comment]
Yes this is a bug that I need to fix with the GUI config file creation. In the meantime, you can manually fix the volume line in the rEFInd config file for the Windows SD card entry.

**Comment by lliamXD383** [comment]
> Yes this is a bug that I need to fix with the GUI config file creation. In the meantime, you can manually fix the volume line in the rEFInd config file for the Windows SD card entry.

how?

**Comment by lliamXD383** [comment]
how

**Comment by CJNiiva** [comment]
I would also like to know, I’m using an USB I created on rufus, on the go install

**Comment by jlobue10** [comment]
On the main repo README page there is a section called "Windiws from Micro SD card..." that goes over this. I will try to fix the GUI bug when I can find time to work on and test the GUI code in up-to-date SteamOS.

**Comment by CJNiiva** [comment]
I figured it out, had to make a label on the efi partition, and add it to the volume on the .conf file

Sendt fra Outlook for iOS<https://aka.ms/o0ukef>
________________________________
Fra: jlobue10 ***@***.***>
Sendt: Sunday, November 3, 2024 3:36:13 PM
Til: jlobue10/SteamDeck_rEFInd ***@***.***>
Kopi: CJNiiva ***@***.***>; Comment ***@***.***>
Emne: Re: [jlobue10/SteamDeck_rEFInd] Invalid loader file! Error: Not found while loading bootmgfw.efi on sd card (Issue #125)


On the main repo README page there is a section called "Windiws from Micro SD card..." that goes over this. I will try to fix the GUI bug when I can find time to work on and test the GUI code in up-to-date SteamOS.

**Comment by jlobue10** [comment]
This should be fixed in 1.4.0 . Closing
